src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -150,62 +127,47 @@ function dismissError(item) {
     }
 }
 
-/**
- * @param {Array} filteredPaymentMethods
- * @param {Boolean} isDefault
- * @returns {Boolean}
- */
-function shouldShowDefaultBadge(filteredPaymentMethods, isDefault = false) {
+function shouldShowDefaultBadge(filteredPaymentMethods: PaymentMethod[], isDefault = false) {
```

Comment:
add return type

Reply:
I think it is good to have return type if it is not void function

Reply:
I agree